docs/explanation/rocks.rst

```diff
@@ -3,22 +3,49 @@
 ROCKs
 =====
 
-ROCKs are container images that are compliant with the `Open Container
-Initiative`_ (OCI) `image format specification <OCI_image_spec_>`_.
-This means that ROCKs are interoperable with other tools that work with
-OCI-compliant images.
+ROCKs are **Ubuntu LTS-based container images** that are designed to meet
+cloud-native software's security, stability, and reliability requirements.
```

Comment:
Does the ubuntu image is one of the more secure and stable?  because for me, the sentence sounds like this

Reply:
i see thanks